Source/Relativity.Test.Helpers/Relativity.Test.Helpers/TestHelper.cs

```diff
@@ -58,31 +62,47 @@ public static IHelper System()
 
 		public IDBContext GetDBContext(int caseID)
 		{
-			//You can create a new DBcontext using kCura.Data.RowDataGeteway until Relativity versions lower than 9.6.85.9
-			//kCura.Data.RowDataGateway.Context context = new kCura.Data.RowDataGateway.Context(SharedTestHelpers.ConfigurationHelper.SQL_SERVER_ADDRESS, string.Format("EDDS{0}", caseID == -1 ? "" : caseID.ToString()), SharedTestHelpers.ConfigurationHelper.SQL_USER_NAME, SharedTestHelpers.ConfigurationHelper.SQL_PASSWORD);
-			//return new DBContext(context);
-
-			//You can create a new DBcontext using DBContextHelper for Relativity versions equal to or greater than 9.6.85.9
-			DbContext context;
+			throw new NotImplementedException();
+		}
 
-			if (_alternateConfig != null)
-			{
-				context = new DbContext(this._alternateConfig.SqlServerAddress, $"EDDS{(caseID == -1 ? "" : caseID.ToString())}", this._alternateConfig.SqlUserName, this._alternateConfig.SqlPassword);
-			}
-			else
+		public Guid GetGuid(int workspaceID, int artifactID)
+		{
+			HttpClient httpClient = GetHttpClient(out var restAddress);
+			string request = "{\"artifactID\":\"@artifactID\",\"workspaceID\":\"@workspaceID\"}";
+			request = request.Replace("@artifactID", artifactID.ToString());
+			request = request.Replace("@workspaceID", workspaceID.ToString());
+			string endpointUrl = restAddress + "/Relativity.REST/api/TestHelpersModule/v1/TestHelpersService/GetGuid";
+			HttpResponseMessage response = MakePostRequest(request, httpClient, endpointUrl);
+			if (!response.IsSuccessStatusCode)
 			{
-				context = new DbContext(SharedTestHelpers.ConfigurationHelper.SQL_SERVER_ADDRESS, $"EDDS{(caseID == -1 ? "" : caseID.ToString())}", SharedTestHelpers.ConfigurationHelper.SQL_USER_NAME, SharedTestHelpers.ConfigurationHelper.SQL_PASSWORD);
+				throw new Exception("Failed to Get Artifact Guid");

```

Comment:
I think we are not supposed to throw normal System Exceptions for stuff like this

Reply:
@calimetikcura Should I change this to a different type of exception?

Reply:
yes. if TestHelpers doesn't have a custom exception, you should create one and start throwing that. This was recommended in the Architecture Lunch and learn. Good catch Tom

Reply:
There are examples in our other tools or helpers for creating custom exceptions.